ui/src/app/pages/workspace/workspace-share.tsx

```diff
@@ -330,15 +270,7 @@ export const WorkspaceShare = fp.flow(withUserProfile())(
             userRoles: resp.items,
             userRolesToChange: [],
           });
-
-          // TODO: should we do anything for failed workflow role revocations?
-          if (resp.failedWorkflowGrants?.length > 0) {
```

Comment:
this is now always false, because it is no longer possible to have failed workflow grants.  RW-8469 will remove this field.